raster.py

```diff
@@ -308,5 +362,115 @@ def cellulose_isoscape_geotiff() -> AmazonGeoTiff:
+def dem_geotiff() -> AmazonGeoTiff:
+  global dem_geotiff_
+  if not dem_geotiff_:
+    dem_geotiff_ = load_named_raster(get_raster_path("dem_pa_brasil_raster.tiff"), "dem",  use_only_band_index=0)
+  return dem_geotiff_
+
+pa_geotiff_ = None
+def pa_geotiff() -> AmazonGeoTiff:
+  global pa_geotiff_
+  if not pa_geotiff_:
+    pa_geotiff_ = load_named_raster(get_raster_path("dem_pa_brasil_raster.tiff"), "pa",  use_only_band_index=1)
+  return pa_geotiff_
+
+krig_means_isoscape_geotiff_ = None
+def krig_means_isoscape_geotiff() -> AmazonGeoTiff:
+  global krig_means_isoscape_geotiff_
+  if not krig_means_isoscape_geotiff_:
+    krig_means_isoscape_geotiff_ = load_named_raster(get_raster_path("uc_davis_d18O_cel_ordinary_random_grouped_means.tiff"), "ordinary_krig_means")
+  return krig_means_isoscape_geotiff_
+
+krig_variances_isoscape_geotiff_ = None
+def krig_variances_isoscape_geotiff() -> AmazonGeoTiff:
+  global krig_variances_isoscape_geotiff_
+  if not krig_variances_isoscape_geotiff_:
+    krig_variances_isoscape_geotiff_ = load_named_raster(get_raster_path("uc_davis_d18O_cel_ordinary_random_grouped_vars.tiff"), "ordinary_krig_vars")
+  return krig_variances_isoscape_geotiff_  
+
+precipitation_regression_isoscape_geotiff_ = None
+def precipitation_regression_isoscape_geotiff() -> AmazonGeoTiff:
+  global precipitation_regression_isoscape_geotiff_
+  if not precipitation_regression_isoscape_geotiff_:
+    precipitation_regression_isoscape_geotiff_ = load_named_raster(get_raster_path("isoscape_fullmodel_d18O_prec_REGRESSION.tiff"), "precipitation_regression_isoscape_geotiff")
+  return precipitation_regression_isoscape_geotiff_
+
+craig_gordon_isoscape_geotiff_ = None
+def craig_gordon_isoscape_geotiff() -> AmazonGeoTiff:
+  global craig_gordon_isoscape_geotiff_
+  if not craig_gordon_isoscape_geotiff_:
+    craig_gordon_isoscape_geotiff_ = load_named_raster(get_raster_path("Iso_Oxi_Stack_mean_TERZER.tif"), "craig_gordon_isoscape_geotiff")
+  return craig_gordon_isoscape_geotiff_
+
+brisoscape_geotiff_ = None
+def brisoscape_geotiff() -> AmazonGeoTiff:
+  global brisoscape_geotiff_
+  if not brisoscape_geotiff_:
+    brisoscape_geotiff_ = load_named_raster(get_raster_path("brisoscape_mean_ISORIX.tif"), "brisoscape_geotiff")
+  return brisoscape_geotiff_
+
+d13C_mean_geotiff_ = None
+def d13C_mean_geotiff() -> AmazonGeoTiff:
+  global d13C_mean_geotiff_
+  if not d13C_mean_geotiff_:
+    d13C_mean_geotiff_ = load_named_raster(get_raster_path("d13C_cel_map_BRAZIL_stack.tiff"), "d13C_mean", use_only_band_index=0)
+  return d13C_mean_geotiff_
+
+d13C_var_geotiff_ = None
+def d13C_var_geotiff() -> AmazonGeoTiff:
+  global d13C_var_geotiff_
+  if not d13C_var_geotiff_:
+    d13C_var_geotiff_ = load_named_raster(get_raster_path("d13C_cel_map_BRAZIL_stack.tiff"), "d13C_var", use_only_band_index=1)
+  return d13C_var_geotiff_
+
+# A collection of column names to functions that load the corresponding geotiffs.
+column_name_to_geotiff_fn = {
```

Comment:
Nit: I think that this dict and the methos above that we use can be encapsulated in a class and that would make this easier to read.